media-sound/cmus/cmus-2.7.1.recipe

```diff
@@ -0,0 +1,91 @@
+SUMMARY="Curses based Music Player"
+DESCRIPTION="
+cmus is a small, fast and powerful console music player for Unix-like operating systems.\
+"
+HOMEPAGE="https://cmus.github.io/"
+SOURCE_URI="https://github.com/cmus/cmus/archive/v2.7.1.tar.gz"
+CHECKSUM_SHA256="8179a7a843d257ddb585f4c65599844bc0e516fe85e97f6f87a7ceade4eb5165"
+LICENSE="GNU GPL v2"
+COPYRIGHT="2005-2015 The cmus Project"
+REVISION="1"
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	cmus$secondaryArchSuffix = $portVersion
+	cmd:cmus$secondaryArchSuffix = $portVersion
+	cmd:cmus_remote$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libcddb$secondaryArchSuffix
+	lib:libFLAC$secondaryArchSuffix
+	lib:libmad$secondaryArchSuffix
+	lib:libmodplug$secondaryArchSuffix
+	lib:libmpcdec$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libmp4v2$secondaryArchSuffix
+	lib:libfaad$secondaryArchSuffix
+	lib:libavformat$secondaryArchSuffix
+	lib:libsamplerate$secondaryArchSuffix
+	lib:libao$secondaryArchSuffix >= 1.1.0
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libncurses$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libcddb$secondaryArchSuffix
+	devel:libFLAC$secondaryArchSuffix
+	devel:libmad$secondaryArchSuffix
+	devel:libmodplug$secondaryArchSuffix
+	devel:libmpcdec$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	devel:libmp4v2$secondaryArchSuffix
+	devel:libfaad$secondaryArchSuffix
+	devel:libavformat$secondaryArchSuffix
+	devel:libsamplerate$secondaryArchSuffix
+	devel:libao$secondaryArchSuffix >= 1.1.0
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gcc${secondaryArchSuffix}
+	cmd:ld${secondaryArchSuffix}
+	cmd:libtoolize
+	cmd:make
+	cmd:m4
+	cmd:pkg_config${secondaryArchSuffix}
+	cmd:sed
+	cmd:find
+	cmd:g++$secondaryArchSuffix
+	cmd:cmake
+	"
+
+PATCHES="cmus-2.7.1-gcc2.patch"
+
+BUILD()
+{
+
+	#	ToDo: We need to set the default output plugin in cmus:
+	#	:set output_plugin=ao
+
+	./configure  \
```

Comment:
Is this an Autotools configure? If not, add a comment saying `not an autotools configure`.